pandas/_libs/tslibs/offsets.pyx

```diff
@@ -4672,6 +4668,9 @@ _dont_uppercase = {
     "qe-oct",
     "qe-nov",
     "ye",
+    "min",
+    "ns",
+    "us",
```

Comment:
can you remove unrelated changes please?

Reply:
If we remove these from _`dont_uppercase` the tests `test_offset_freqstr()` and `test_rule_code()` in `pandas/tests/tseries/offsets/test_offsets.py` will fail. The reason: we uppercase names of offsets class in `_get_offset()`.  In PR #56346 where we deprecate uppercasing/lowercasing we will remove them from `_dont_uppercase`

Reply:
ah, thanks - so, they're not currently unused then? but will become unused once https://github.com/pandas-dev/pandas/pull/56346 is enforced? should this PR wait until after that one then?

Reply:
I checked in PR #56346, we should keep `"min", "us", "ns"` in the list _dont_uppercase to pass these tests.
On the other hand, we have all lowercase offsets frequencies in this list (`'h', 's'`, etc.), and it looks more clear then have these entries (`"min": "min", "us": "us"`, etc.) in dict `_lite_rule_alias`.

Reply:
thanks for explaining

sorry for not having got round to this yet, was quite this week with a client project - but I'll get to it, thanks for looking at it all so carefully!

Reply:
thank you for helping me with my PRs!